src/index.ts

```diff
@@ -21,32 +21,49 @@ export class TestxServer {
   private schema: string;
   private server: Server;
   private serverUrl: string;
-  private dbConnection: Knex;
+  private backendContext: { [id: string]: any };
 
   constructor(schema: string) {
     this.schema = schema;
   }
 
   public async start() {
-    const { server, url } = await this.generateServer();
+    if (!this.backendContext) { await this.generateBackend(); }
```

Comment:
This is good, but I would prefer to put this logic in a public `bootstrap()` method so that it could be also called separately before `start()`
```suggestion
    await this.bootstrap();
```

Reply:
@b1zzu I like the idea of putting that logic in another method, but why don't we use a private method?

Reply:
Because the bootstrap method is the more expensive some user may want to put it in a separate test phase with a different timeout, but I don't have a real use case now, so making it private is fine and we can always make it public in the future

Reply:
I get it, so I'll make it public.
We can put in this method the call to `generateServer` and the storage of `dbConnection` and express server, wdyt?

Reply:
I'm gonna make a commit with these changes, so we can see the best way :)